tests/unit/api/test_feature_list.py

```diff
@@ -610,11 +610,6 @@ def test_list_versions(saved_feature_list):
                     flist_1.version,
                     saved_feature_list.version,
                 ],
-                "feature_list_namespace_id": [
-                    flist_2.feature_list_namespace_id,
-                    flist_1.feature_list_namespace_id,
-                    saved_feature_list.feature_list_namespace.id,
-                ],
```

Comment:
For `FeatureList.list_versions()`, we only remove `feature_list_namespace_id` and keep `num_feature`.